<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserBidDialog</class>
 <widget class="QDialog" name="UserBidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_Instruction">
       <property name="text">
        <string>Instruction</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="gb_Suit">
         <property name="title">
          <string>Select Suit/Bid</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="rb_Spades">
            <property name="text">
             <string>Spades</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_Clubs">
            <property name="text">
             <string>Clubs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_Diamonds">
            <property name="text">
             <string>Diamonds</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_Hearts">
            <property name="text">
             <string>Hearts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_NoTrump">
            <property name="text">
             <string>No Trump</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_Nellow">
            <property name="text">
             <string>Nellow</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_OpenNellow">
            <property name="text">
             <string>Open Nellow</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_DoubleNellow">
            <property name="text">
             <string>Double Nellow</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="QPushButton" name="pb_Pass">
           <property name="text">
            <string>Pass</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pb_Bid">
           <property name="text">
            <string>Bid</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="gb_NumTricks">
           <property name="title">
            <string># of Tricks</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="rb_6Tricks">
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_7Tricks">
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_8Tricks">
              <property name="text">
               <string>8</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_9Tricks">
              <property name="text">
               <string>9</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_10Tricks">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
